R/transformCounts.R

```diff
@@ -392,7 +394,7 @@ setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
         ref_dimnames <- list(var_names, reference_name)
         }
     # Reference sample as NAs or with symbols that are specified by user
-    reference_sample <- matrix(ref_vals, nrow = nrow, ncol = ncol,  
+    reference_sample <- matrix(reference, nrow = nrow, ncol = ncol,  
```

Comment:
should it be more clear `reference.value` or `ref.value`

Reply:
That name is chose because vegan::decostand uses it https://rdrr.io/cran/vegan/man/decostand.html

Reply:
ok